projects/schematics/src/add/index.ts

```diff
@@ -48,13 +48,12 @@ function setProjectSettings(options: NgAddOptions) {
 }
 
 // Checks if a version of Angular is compatible with current or next
-function getVersion(ngVersion: string, clrVersion: string): string {
-  const diff = 6; // Number disparity between Angular and Clarity, this works as long as we stay in sync with versioning
+export function getVersion(ngVersion: string, clrVersion: string): string {
+  const diff = 0; // Number disparity between Angular and Clarity, this works as long as we stay in sync with versioning
```

Comment:
So we need to do more here, if the Angular version is less than 12, we should use the original logic. That way it will still work on Angular 11 or earlier projects. The CLI will always download the 'latest' version of a package to run `ng-add` commands, so we have to retain backwards compatability.